tests/test_logging.py

```diff
@@ -19,20 +19,23 @@ def test_default_handler(capsys):
 
     # We need to reconstruct our default handler to properly capture stderr.
     optuna.logging._reset_library_root_logger()
+    optuna.logging.set_verbosity(optuna.logging.INFO)
+
     library_root_logger = optuna.logging._get_library_root_logger()
+    library_root_logger.propagate = False
```

Comment:
To avoid interference with the TensorFlow's logger, I changed to not propagate Optuna's log messages to the root logger.